src/vs/workbench/contrib/chat/browser/actions/chatAccessibilityHelp.ts

```diff
@@ -13,22 +13,26 @@ import { IChatWidgetService } from 'vs/workbench/contrib/chat/browser/chat';
 import { IAccessibleViewService } from 'vs/workbench/contrib/accessibility/browser/accessibleView';
 import { InlineChatController } from 'vs/workbench/contrib/inlineChat/browser/inlineChatController';
 
+
+const inputBox = localize('chat.requestHistory', 'In the input box, use up and down arrows to navigate your request history. Edit input and use enter or the submit button to run a new request.');
+
 export function getAccessibilityHelpText(accessor: ServicesAccessor, type: 'chat' | 'inline'): string {
 	const keybindingService = accessor.get(IKeybindingService);
 	const content = [];
 	if (type === 'chat') {
-		content.push(localize('chat.overview', 'Chat responses will be announced as they come in. A response will indicate the number of code blocks, if any, and then the rest of the response.'));
-		content.push(localize('chat.requestHistory', 'In the input box, use up and down arrows to navigate your request history. Edit input and use enter to run a new request.'));
-		content.push(descriptionForCommand('chat.action.focus', localize('workbench.action.chat.focus', 'The Focus Chat command ({0}) focuses the chat request/response list, which can be navigated with UpArrow/DownArrow.',), localize('workbench.action.chat.focusNoKb', 'The Focus Chat List command focuses the chat request/response list, which can be navigated with UpArrow/DownArrow and is currently not triggerable by a keybinding.'), keybindingService));
+		content.push(localize('chat.overview', 'The chat view is comprised of an input box and a request/response list. The input box is used to make requests and the list is used to display responses.'));
```

Comment:
composed of?

Reply:
Comprised = made of, I think this works

Reply:
I think you would say "comprises". But not a big deal